docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -1,84 +1,144 @@
 = Using the Data Migration Tool
-:description: This guide explains how to migrate data from Hazelcast 4.x or 5.x clusters to newer 5.x clusters.
+:description: You can use the Data Migration Tool (DMT) to migrate your data from version 4.x or 5.x Open Source and Enterprise Edition Hazelcast clusters when upgrading to 5.3.x versions of Enterprise Edition, or moving to the latest {hazelcast-cloud} release.
 
-[blue]*Hazelcast Enterprise Feature*
+{description} 
 
-{description} The purpose of Data Migration Tool (DMT) is to migrate data from your Open Source or Enterprise clusters, to either a newer Enterprise cluster or a cluster on Hazelcast Viridian.
+NOTE: The DMT migrates your data for maps and replicated maps only. Metadata is not migrated.
```

Comment:
Metadata of maps are migrated but metadata of replicated maps are not migrated

Reply:
Updated Note